README.md

```diff
@@ -103,15 +103,26 @@ To maintain records of the API's dependencies,
 command:
 
 ```bash
-curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+curl -sSL https://install.python-poetry.org | python3 -
 ```
 
-The installation requires `~/.poetry/env` to be refreshed for changes to be applied.
-Open a new terminal or execute the following command to ensure the installation is
-completed smoothly:
+The installation requires the following to be added to your `~/.bashrc` file so the installation folder is on your path.
+```bash
+export PATH= "~/.local/bin:$PATH"
```

Comment:
There is a space between `=` and `"` that shouldn't be there. Could this be removed?

Reply:
Has been removed